.travis.yml

```diff
@@ -20,11 +20,9 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.8.*@dev
     - php: 5.6
-      env: SYMFONY_VERSION="3.0.x-dev as 2.8"
+      env: SYMFONY_VERSION="3.0.x-dev as 2.8" PHPUNIT_FLAGS="--exclude-group legacy"
```

Comment:
btw, proper skipping will be necessary once 3.0 gets released as stable too, as it would be used by default in builds